### PR TITLE
ADCM-2465 add unique constraint for name and display_name in role

### PR DESCRIPTION
**Pull request by SofiaTal**

### CODE REVIEW

File: python/rbac/endpoints/role/views.py [hunk 1]
```diff
@@ -29,9 +30,20 @@ class RoleChildSerializer(BaseRelatedSerializer):
 class RoleSerializer(FlexFieldsSerializerMixin, serializers.ModelSerializer):
     url = serializers.HyperlinkedIdentityField(view_name='rbac:role-detail')
     child = RoleChildSerializer(many=True, required=False)
-    name = serializers.RegexField(r'^[^\n]*$', max_length=160, required=False)
+    name = serializers.RegexField(r'^[^\n]*$', max_length=160, required=False, allow_blank=True)
     display_name = serializers.RegexField(r'^[^\n]*$', max_length=160, required=True)
 
+    def validate_display_name(self, display_name):
+        try:
```
Comment (nagarin) [inline python/rbac/endpoints/role/views.py:37]: If Role.objects.get(display_name=display_name, type=RoleTypes.role).exists():
    raise ...
return display_name

**Review by nagarin — CHANGES_REQUESTED** [review]

**Review by nagarin — APPROVED** [review]

**Comment by acmnu** [comment]
Why we need this? Do we have such requirements somewhere?

**Review by nagarin — APPROVED** [review]

**Comment by nagarin** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @nagarin! Build has been launched!

**Pull request merged**